backend/app/Services/Domain/Order/OrderCreateRequestValidationService.php

```diff
@@ -186,7 +188,16 @@ private function validateTicketQuantity(int $ticketIndex, array $ticketAndQuanti
     {
         $totalQuantity = collect($ticketAndQuantities['quantities'])->sum('quantity');
         $maxPerOrder = (int)$ticket->getMaxPerOrder() ?: 100;
-        $minPerOrder = (int)$ticket->getMinPerOrder() ?: 1;
+
+        $capacityMaximum = $this->availableTicketQuantities
+            ->ticketQuantities
+            ->where('ticket_id', $ticket->getId())
+            ->map(fn(AvailableTicketQuantitiesDTO $price) => $price->capacities)
+            ->flatten()
+            ->min(fn(CapacityAssignmentDomainObject $capacity) => $capacity->getCapacity());
+
+        # if there are fewer tickets available than the configured minimum, we allow less than the minimum to be purchased
+        $minPerOrder = min((int)$ticket->getMinPerOrder() ?: 1, $capacityMaximum ?: $maxPerOrder);
```

Comment:
I believe we'll also need to take the ticket's remaining quantity into account. Especially if there is no Capacity Assignment associated with the ticket.

```php
        $ticketAvailableQuantity = $this->availableTicketQuantities
            ->ticketQuantities
            ->first(fn(AvailableTicketQuantitiesDTO $price) => $price->ticket_id === $ticket->getId())
            ->quantity_available;
```